### DISCUSSION TITLE
when will xquic support windows?

### DISCUSSION THREAD

**Discussion by GreenValue** [alibaba/xquic#76] [General]
### Before request a new feature

- [X] I have searched the [issues](https://github.com/alibaba/xquic/issues) of this repository and believe that this is not a duplicate.

### Describe the feature you'd like supported

when will xquic support windows?
I want to use xquic to trasnfer file in windows exe.

**Comment by ruiqizhou** [comment]
Not in the short term. Please see our [milestones](https://github.com/alibaba/xquic/milestones) for what we are currently doing.
